src/main/kotlin/fi/metatavu/timebank/api/impl/translate/TimeEntryTranslator.kt

```diff
@@ -11,14 +11,15 @@ class TimeEntryTranslator: AbstractTranslator<TimeEntry, fi.metatavu.timebank.mo
 
     override fun translate(entity: TimeEntry): fi.metatavu.timebank.model.TimeEntry {
         return fi.metatavu.timebank.model.TimeEntry(
-            forecastId = entity.forecastId!!,
+            entryId = entity.entryId,
+            forecastId = entity.forecastId,
             person = entity.person!!,
             internalTime =  entity.internalTime!!,
-            projectTime = entity.projectTime!!,
+            billableProjectTime = entity.billableProjectTime!!,
+            nonBillableProjectTime = entity.nonBillableProjectTime!!,
             date = entity.date!!,
             createdAt = entity.createdAt!!,
             updatedAt = entity.updatedAt!!,
-            id = entity.entryId,
```

Comment:
No id at all?

Reply:
Renamed to entryId. It has been named entryId in the api spec since the beginning of this project and this was particular edit was made to keep the naming consistent.

Reply:
Although I think it would be best to call the attribute with PK constraint id instead of entryId.. So I Should propably change that?